gamemodes/terrortown/gamemode/client/cl_tbuttons.lua

```diff
@@ -163,25 +167,36 @@ function TBHUD:Draw(client)
 		pos = ent:GetPos()
 		scrpos = pos:ToScreen()
 
-		if util.IsOffScreen(scrpos) or not ent:IsUsable() then continue end
+		local percent = math.Clamp((ent:GetNextUseTime() - CurTime()) / ent:GetDelay(), 0, 1)
+		local end_angle = 360 * percent
+
+		-- if util.IsOffScreen(scrpos) or not ent:IsUsable() then continue end
```

Comment:
What about this one?

Reply:
added back the offscreen check but not the usable check, since that really isn't a concern anymore